Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -9,7 +9,10 @@ Due to package verification, the latest version below is the unpublished version
 however, it has to be formatted properly to pass verification tests.
 
 ## [Unreleased] - yyyy-mm-dd
-- Fixed Composite binding isn't triggered after ResetDevice() called during Action handler [ISXB-746](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-746)
+
+### Fixed
+- Avoid potential crashes from NullReferenceException in FireStateChangeNotifications.
```

Comment:
Minor: Use code formatting quotes, e.g. `NullReferenceException` in `FireStateChangeNotifications`.